app/controllers/sites/documentations_controller.rb

```diff
@@ -1,6 +1,5 @@
 class Sites::DocumentationsController < Sites::BaseController
   # see ForumsController
-  skip_before_action :activate_menu_site_or_settings
```

Comment:
@Martouta this method does not exist so rails 5.x will just throw an error

Maybe the content was there but due to many rebases on master it disappeared because already included in master